packages/ndla-ui/src/Article/Article.tsx

```diff
@@ -186,6 +197,12 @@ export const Article = ({
       <ArticleWrapper modifier={modifier} id={id} ref={articleRef}>
         <LayoutItem layout="center">
           {accessMessage && <ArticleAccessMessage message={accessMessage} />}
+
+          {messagebox && (
+            <MSGboxWrapper>
+              <MessageBox links={messageBoxLinks}>{messageForBox}</MessageBox>
+            </MSGboxWrapper>
+          )}
```

Comment:
Du treng ikkje egen messagebox-prop om du berre sjekker om teksten finnes.
```suggestion
          {messages.messageBox && (
            <MSGboxWrapper>
              <MessageBox links={messageBoxLinks}>{messages.messageBox}</MessageBox>
            </MSGboxWrapper>
          )}
```